contracts/VotiumBribesProcessor.sol

```diff
@@ -228,9 +235,8 @@ contract VotiumBribesProcessor is CowSwapSeller {
         uint256 toEmit = BADGER.balanceOf(address(this));
         require(toEmit > 0);
 
-        BADGER.safeTransfer(BADGER_TREE, toEmit);
-
-        emit TreeDistribution(address(BADGER), toEmit, block.number, block.timestamp);
+        BADGER.safeApprove(address(HARVEST_FORWARDER), toEmit);
+        HARVEST_FORWARDER.distribute(address(BADGER), toEmit, address(BVE_CVX));
```

Comment:
There should be two distributions here based on a percent split we can modify:
- bveCVX distribution
- bveCVX LP distribution

Details of the implementation are here:
https://github.com/Badger-Finance/badger-multisig/blob/56bb76914b9e665997702fe2e82e08a4ffe1599f/scripts/emissions/bribes_emissions.py#L18-L23